pkg/cloud/aws/actuators/machine_scope.go

```diff
@@ -33,17 +35,22 @@ type MachineScopeParams struct {
 	Cluster *clusterv1.Cluster
 	Machine *clusterv1.Machine
 	Client  client.ClusterV1alpha1Interface
+	Logger  logr.Logger
 }
 
 // NewMachineScope creates a new MachineScope from the supplied parameters.
 // This is meant to be called for each machine actuator operation.
 func NewMachineScope(params MachineScopeParams) (*MachineScope, error) {
-	scope, err := NewScope(ScopeParams{AWSClients: params.AWSClients, Client: params.Client, Cluster: params.Cluster})
+	scope, err := NewScope(ScopeParams{
+		AWSClients: params.AWSClients,
+		Client:     params.Client, Cluster: params.Cluster,
+		Logger: params.Logger,
```

Comment:
Should the `WithName` context be added here instead of below?

Reply:
No, otherwise machine will come before the APIVersion